### PR TITLE
TCM-129 | Custom Loading HUD

### PR DESCRIPTION
**Pull request by vespertineSiren**
### Summary

- Use generic yellow loading dots until replaced by TRX loader. 

### Issue

[TCM-129](https://hyfnla.atlassian.net/browse/TCM-129)

### Video/Screenshot

**Current setup that was pushed**
```
           android:layout_width="90dp"
           android:layout_height="90dp"
           app:lottie_repeatMode="reverse"
```

https://user-images.githubusercontent.com/27742105/120082030-7a454a00-c075-11eb-8798-3c1f8f3f82a1.mov

I found this to be a little small and the repeat didn't make sense so I made it larger and repeat as a possible alternative. 
```
           android:layout_width="180dp"
           android:layout_height="180dp"
           app:lottie_repeatMode="repeat"
```

https://user-images.githubusercontent.com/27742105/120082069-b8db0480-c075-11eb-9788-0a8421e28631.mov

### CODE REVIEW

File: app/src/main/res/layout/layout_hud.xml [hunk 1]
```diff
@@ -13,12 +14,19 @@
 
     <com.trx.consumer.common.CommonView
         style="@style/MatchParentEdgesViewStyle"
-        android:layout_width="45dp"
-        android:layout_height="45dp">
+        android:layout_width="wrap_content"
```
Comment (VivekV95) [inline app/src/main/res/layout/layout_hud.xml:17]: Looks like iOS has the loading animation nested in another view 

https://user-images.githubusercontent.com/39810816/120084403-86410400-c095-11eb-91b3-a7eec381a40d.mov

**Review by VivekV95 — CHANGES_REQUESTED** [review]
Had a question

File: app/src/main/res/layout/layout_hud.xml (hunk 1 above)
Comment (vespertineSiren) [reply app/src/main/res/layout/layout_hud.xml:17]: Yeah Onur mentioned that as well. Will do

**Comment by onodude** [comment]
please copy and paste the specs from xpass [here](https://github.com/hyfn/xpass-android/blob/develop/app/src/main/res/layout/layout_hud.xml)

**Review by onodude — APPROVED** [review]

**Pull request merged**